Release/tests/functional/http/client/compression_tests.cpp

```diff
@@ -206,7 +206,7 @@ SUITE(compression_tests)
         std::vector<uint8_t> cmp_buffer(buffer_size);
         size_t cmpsize = buffer_size;
         size_t csize = 0;
-        operation_result r = {0};
+        operation_result r = {0, 0, false};
```

Comment:
cleanly initialise all the members of the structure

Reply:
I think `operation_result r{};` might be a better fix, but I don't know exactly what warnings you're working around.

Reply:
Yes, that also fixes it.  The warning is:
```
.../cpprestsdk/Release/tests/functional/http/client/compression_tests.cpp:209:32: warning: missing initializer for member ‘web::http::compression::operation_result::output_bytes_produced’ [-Wmissing-field-initializers]
  209 |         operation_result r = {0};
      |                                ^

```